develop/tutorials/articles/120-customizing/03-overriding-osgi-services/01-examining-an-service-to-override.markdown

```diff
@@ -0,0 +1,190 @@
+  and
+  [Maven Central](https://search.maven.org/)
+  (for Liferay and non-Liferay artifact Javadocs). 
+
+$$$
+
+## Step 2: Copy the Existing Service Name [](id=step-2-copy-the-existing-service-name)
+
+If you want to invoke the existing service along with your custom service, get the existing service name. 
+
+The `src:info` result's Component Configuration section lists the existing
+service's fully qualified name. For example, the
+`OverrideMyServiceReferencePortlet` component's references `_someService` is
+bound to a service component whose fully qualified name is
+`override.my.service.reference.service.impl.SomeServiceImpl`. 
+
+    Component Configuration:
+    ...
+    SatisfiedReference: _someService
+      ...
+      Bound to:        6840
+          Properties:
+            ...
+            component.name = override.my.service.reference.service.impl.SomeServiceImpl
+
+**Copy the `component.name`** so you can reference the service in your 
+[custom service](TODO).
+
+Here's an example of referencing the service above. 
+
+    @Reference  (
+        target = "(component.name=override.my.service.reference.service.impl.SomeServiceImpl)"
+    )
+    private SomeService _defaultService;
+
+## Step 3: Gather Reference Configuration Details (if reconfiguration is needed) [](id=step-3-gather-reference-configuration-details-if-reconfiguration-is-needed)
+
+The service reference's policy and policy option determine a component's conditions for adopting a particular service.
+
+- If the reference's policy option is `greedy`, it binds to the matching, 
+highest ranking service right away. The reference need not be reconfigured to
+adopt your service. 
+
+- If policy is `static` and its policy option is `reluctant`, however, the  
+component requires one of the following conditions to switch from using the
+existing service it's referencing to using the matching, highest ranking service
+(i.e., you'll rank your custom service highest):
+
+    1. The component is reactivated
+    2. The component's existing referenced service is unavailable
+    3. The component's reference is modified so that it does not match the existing service but matches your service
+
+[Reconfiguring the reference](/develop/tutorials/-/knowledge_base/7-1/reconfiguring-components-to-use-your-service)
+is straightforward and can be the quickest way for the component to adopt a new
+service. 
+
+**Gather these details:** 
+
+- *Component name:* Find this at *Component Description* &rarr; *Name*. For example,
+
+        Component Description:
+            Name: override.my.service.reference.portlet.OverrideMyServiceReferencePortlet
+            ...
+
+- *Reference name:* The *Reference* value (e.g., `Reference: _someService`).
+
+- *Cardinality:* Number of service instances the reference can bind to. 
+
++$$$
+
+**Note**: OSGi Configuration Admin makes all Declarative Services components
+configurable, even if they don't explicitly declare
```

Comment:
> OSGi Configuration Admin makes all Declarative Services components
configurable

Can you change this line to:
`Declarative Services makes all components configurable through OSGi Configuration Admin`

This would be more correct.